ci-operator/step-registry/gcp/provision/xpn-minpt-serviceaccount/gcp-provision-xpn-minpt-serviceaccount-commands.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+echo "$(date -u --rfc-3339=seconds) - Enabling the IAM service account of minimal permissions for deploying OCP cluster into GCP shared VPC..."
+
+# For now using the same service account as the user-tags testing, 
+# but it is possible to use another service account in future. 
+cp "${CLUSTER_PROFILE_DIR}/user_tags_sa.json" "${SHARED_DIR}/xpn_minpt_sa.json"
```

Comment:
`minpt` means `minimal permissions` ?

Reply:
Sorry, "minpt" means mimimal permissions for credentials mode being Passthrough. Shall I update it? And your suggestion? Thanks!

Reply:
my suggestion is `xpn_min_perm_passthrough.json`